Riot/Categories/MXSession+Riot.m

```diff
@@ -52,15 +52,15 @@ - (NSUInteger)vc_missedDiscussionsCount
 - (BOOL)vc_isE2EByDefaultEnabledByHSAdmin
 {
     BOOL isE2EByDefaultEnabledByHSAdmin = YES;
-    
+
```

Comment:
MXSession.vc_isE2EByDefaultEnabledByHSAdmin should be removed to avoid to have wellknown data available from several places.

Reply:
It would be better but with ObjC we don't have nullable boolean. So if `homeserverWellknown` is nil it will return false but this value is intended to be true if there is nothing set.

Reply:
Ok. Then, this should apply for any values. `vc_jitsiPreferredServerURL` should be here.
The widget manager should not handle the default value to `BuildSettings.jitsiServerUrl`.